renga/api/repository.py

```diff
@@ -149,15 +150,35 @@ def init_repository(
         self, name=None, force=False, use_external_storage=True
     ):
         """Initialize a local Renga repository."""
-        self.renga_path.mkdir(parents=True, exist_ok=force)
-
         path = self.path.absolute()
         if force:
+            self.renga_path.mkdir(parents=True, exist_ok=force)
             if self.git is None:
                 self.git = GitRepo.init(str(path))
         else:
-            assert self.git is None, 'Git repo already exists.'
-            self.git = GitRepo.init(str(path))
+            if self.renga_path.exists():
+                raise UsageError(
+                    'You are trying to initialize a Renga '
```

Comment:
The description should be added to `renga.cli.init`. It is specific to the command line interface, but here we are implementing generic API for Renga repository.